UI/window-basic-settings.cpp

```diff
@@ -4258,6 +4298,8 @@ void OBSBasicSettings::on_buttonBox_clicked(QAbstractButton *button)
 
 	if (val == QDialogButtonBox::ApplyRole ||
 	    val == QDialogButtonBox::AcceptRole) {
+		if (!QueryAllowedToClose())
+			return;
```

Comment:
Prefer a blank line after non-enclosed `if` statement.

Reply:
Added (for both occurrences).